pkg/liqoctl/rest/identity/utils.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019-2024 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package identity
+
+import (
+	"fmt"
+	"os"
+
+	"k8s.io/cli-runtime/pkg/printers"
+
+	authv1alpha1 "github.com/liqotech/liqo/apis/authentication/v1alpha1"
+)
+
+// output implements the logic to output the generated Configuration resource.
+func (o *Options) output(conf *authv1alpha1.Identity) error {
```

Comment:
identity and not configuration, both in the comment and in the function argument